src/features/user/TreeMapper/Import/components/PlantingLocation.tsx

```diff
@@ -154,8 +156,8 @@ function PlantedSpecies({
 interface Props {
   handleNext: () => void;
   userLang: string;
-  plantLocation: PlantLocation | null;
-  setPlantLocation: Function;
+  // plantLocation: PlantLocation | null;
```

Comment:
Why is it commented out instead of being removed?

Reply:
Removed